hack/go-install.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -uo pipefail
+
+DEST_DIR=${1:-}
+CMD=${2:-}
+GO_INSTALL_LOC=${3:-}
+
+if [[ ("$DEST_DIR" == "" || "$CMD" == "" || "$GO_INSTALL_LOC" == "") ]]; then
```

Comment:
Curious, as I am not sure, do the brackets make any difference here?

Reply:
I think we can remove them.